src/coordinator/index.spec.js

```diff
@@ -388,6 +388,57 @@ describe('Coordinator', function() {
     });
   });
 
+  describe('getUsersByOrganization', function() {
+    context('the organization ID is provided', function() {
+      let expectedOrganizationId;
+      let organizationUsers;
+      let promise;
+      let request;
+
+      beforeEach(function() {
+        expectedOrganizationId = faker.random.uuid();
+
+        organizationUsers = fixture.buildList(
+          'contxtUser',
+          faker.random.number({ min: 1, max: 10 })
+        );
+
+        request = {
+          ...baseRequest,
+          get: this.sandbox.stub().resolves(organizationUsers)
+        };
+
+        const coordinator = new Coordinator(baseSdk, request);
+        coordinator._baseUrl = expectedHost;
+
+        promise = coordinator.getUsersByOrganization(expectedOrganizationId);
+      });
+
+      it('gets the user list from the server', function() {
+        expect(request.get).to.be.calledWith(
+          `${expectedHost}/organizations/${expectedOrganizationId}/users`
+        );
+      });
+
+      it('returns the list of users by requested organization', function() {
```

Comment:
Need to add a test about the returned users being formatted -- you can find something similar here: https://github.com/ndustrialio/contxt-sdk-js/blob/master/src/coordinator/index.spec.js#L368 (except subbing out applications for users -- note that the stub for `get` returns one object and `toCamelCase` returns another object)